ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -185,69 +223,98 @@ const POPULATIONS = [
   {
     field: 'gnomad_mito',
     fieldTitle: 'gnomAD mito',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
     precision: 3,
-    urls: { [GENOME_VERSION_38]: 'gnomad.broadinstitute.org' },
-    queryParams: { [GENOME_VERSION_38]: 'dataset=gnomad_r3' },
+    ...GNOMAD_URL_INFO,
+  },
+  {
+    field: 'gnomad_mito_heteroplasmy',
+    fieldTitle: 'gnomAD mito',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
+    precision: 3,
+    ...GNOMAD_URL_INFO,
   },
   {
     field: 'helix',
     fieldTitle: 'Helix mito',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
+    precision: 3,
+  },
+  {
+    field: 'helix_heteroplasmy',
+    fieldTitle: 'Helix mito',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
     precision: 3,
   },
 ]
 
+const pushDefaultKey = (acc, key, value) => {
+  if (!acc[key]) {
+    acc[key] = []
+  }
+  acc[key].push(value)
+}
+
 const Frequencies = React.memo(({ variant }) => {
   const { populations = {} } = variant
+  const callsetHetPop = populations.callset_heteroplasmy
+  const isMito = callsetHetPop && callsetHetPop.af !== null && callsetHetPop.af !== undefined
+  const popupDetailsBySection = POPULATIONS.reduce((acc, pop) => {
```

Comment:
I think it would be better for the section constants to define their own display logic instead of having so much hardcoded conditional if/else render logic. Also, since we always show different sections for mito/ non-mito I think we should explicitly use different section constants for those instead of if/else logic. I also think it would be neater to use one constant rather than separate constants for the sections and the population sections. So something more like

```
const DETAIL_SECTIONS = [{
  name: 'Global AFs', 
  hasDetail: pop => pop. filter_af && pop. filter_af !== pop.af, 
  display: pop => ([{ value: pop.af.toPrecision(3) }]),
}, 
...
]
MITO_DETAIL_SECTIONS =  [{
  name: HET_SECTION, 
  hasDetail: pop => pop. ac || pop.max_hl 
  display: pop => ([
    pop.ac && { value: `${pop.ac} out of ${pop.an}` },
    pop.max_hl && { subtitle: ' max observed heteroplasmy', value: pop.max_hl.toPrecision(3) },
  ].filter(val => val)),
}, 
...

const sections = (isMito ? MITO_HOVER_SECTIONS : HOVER_SECTIONS).map(({ hasDetail, ...section }) => ({
  ...section,
  populations: POPULATIONS.filter(pop => (pop.section === undefined || pop.section === section.title) &&  populations[pop.field] && hasDetail(populations[pop.field]))
}), []).filter(({ populations }) => populations.length)
...

    {sections.reduce((acc, section, i) => ([
          ...acc,
          (i > 0) && <VerticalSpacer key={section.name} height={5} />,
          <Popup.Header key={`${section.name}_header`} content={section.name} />,
          ...section.populations.reduce((acc, pop) => ([
             ...acc, ...section.display(populations[pop.field]).map(({ subtitle, value }) => (
              <Popup.Content key={`${section.name}-${pop.field}${subtitle}`}>
                {`${pop.fieldTitle}${subtitle || ''}: ${value`}
              </Popup.Content>
            ))]), []),
        ]), [])}
```

Reply:
Using section constants is a good idea.